x-pack/plugins/index_management/public/application/sections/home/index_list/details_page/details_page.tsx

```diff
@@ -36,78 +41,107 @@ import { DetailsPageMappings } from './details_page_mappings';
 import { DetailsPageOverview } from './details_page_overview';
 import { DetailsPageSettings } from './details_page_settings';
 
-const defaultTabs = [
+const defaultTabs: IndexDetailsTab[] = [
   {
     id: IndexDetailsSection.Overview,
     name: (
       <FormattedMessage id="xpack.idxMgmt.indexDetails.overviewTitle" defaultMessage="Overview" />
     ),
+    renderTabContent: (indexName: string, index: Index) => (
+      <DetailsPageOverview indexDetails={index} />
+    ),
+    order: 10,
```

Comment:
Just a question: why are the default order numbers multiples of 10? This confused me a bit when I tried to change the order of the test tab and it wasn't ordered as I expected, so I guess it might confuse people who are adding tabs in the future. If there is any reason to keep the default order numbers as multiples of 10, it might help to add some documentation/comments for this.

Reply:
Good point, I'll add documentation in a follow up PR and explain the order property.  The reason to use multiples of 10 is to be able to place new tabs at any place in the order. This allows for even several new tabs to be placed before or between the existing ones. The new tabs will be added by other plugins, so that will not change our code and will give them flexibility.